src/kde_analysis/iterative_kde_m1m2chieff.py

```diff
@@ -355,6 +361,8 @@ def get_kde_obj_eval(sample, bs_weights, rescale_arr, alpha, input_transf=('log'
             vt_val[i] = sensitivity(m1_val[i], m2_val[i], chieff=chieff_val[i])
         vth5file.create_dataset(k, data=np.array(vt_val))
 
+    
```

Comment:
remove these lines

Reply:
I did not mean remove the VT calculation. I mean remove the extra print statements which we do not want to have when running the code in production